src/main/resources/config/liquibase/changelog/20231120095701_changelog.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="andreas (generated)" id="1700470627638-3">
+        <createTable tableName="math_exercise_details">
+            <column name="example_solution" type="VARCHAR(255)"/>
```

Comment:
Not sure what the example_solution will contain, but I'd be surprise if 255 chars are enough here.